docs/examples.md

```diff
@@ -11,8 +11,135 @@ The types generated by openapi-typescript are universal, and can be used in a va
 
 Fetching data can be done simply and safely using an **automatically-typed fetch wrapper**:
 
-- [openapi-fetch](/openapi-fetch/) (recommended)
-- [openapi-typescript-fetch](https://www.npmjs.com/package/openapi-typescript-fetch) by [@ajaishankar](https://github.com/ajaishankar)
+<details>
+<summary><a href="/openapi-fetch/">openapi-fetch</a> (recommended)</summary>
+
+::: code-group
+
+```ts [test/my-project.ts]
+import createClient from "openapi-fetch";
+import type { paths } from "./my-openapi-3-schema"; // generated by openapi-typescript
+
+const client = createClient<paths>({ baseUrl: "https://myapi.dev/v1/" });
```

Comment:
❤️ These little improvements go a long way, thank you!

Reply:
Glad you appreciate the changes. Since its the "Example" page I thought I add some to the "Data fetching" section :)